.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+**Description**
+
+<!--- Describe your changes in detail -->
+
+**Related Issue**
+
+<!--- Please link to the issue here -->
+
+**Motivation and Context**
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```

Comment:
All PRs should fix (or partially fix) a related issue, which should address the "motivation and context".

```suggestion
**Description**

<!--- Link to the open issue this addresses -->
Fixes #

<!--- Describe your changes in detail -->
```